app/components/participant-creator/template.hbs

```diff
@@ -100,23 +91,33 @@
                     </tr>
                 </thead>
                 <tbody>
-                    {{#each generatedParticipants as |id index|}}
-                        <tr>
-                            <td>
-                                {{index}}
-                            </td>
+                    <tr>
+                        <td>
+                            {{exampleId}}
+                        </td>
+                        <td>
+                            {{studyId}}
+                        </td>
+                        {{#each extra as |item|}}
                             <td>
-                                {{id}}
+                                {{item.value}}
                             </td>
-                            {{#each extra as |item|}}
-                                <td>
-                                    {{item.value}}
-                                </td>
-                            {{/each}}
-                        </tr>
-                    {{/each}}
+                        {{/each}}
+                    </tr>
                 </tbody>
             </table>
         </div>
     </div>
 </div>
+
+{{#if invalidStudyId}}
```

Comment:
This seems like a question of validation- why not use ember-cp-validations instead, to display the error in the context of where it's being entered? I'm not clear on why this is done via a modal.

As a matter of fact, I've already started adding validations to this exact form in my own overlapping work, so this would make for more consistent UI.
https://github.com/CenterForOpenScience/experimenter/pull/120/files#diff-354dada75ae7fef5f7def4c72524e3a5R20


See also style guide advice on:
Validations: https://centerforopenscience.github.io/osf-style/ui.html#form-validation
Modals: https://centerforopenscience.github.io/osf-style/ui.html#modals